applications/minotari_ledger_wallet/comms/src/ledger_wallet.rs

```diff
@@ -39,12 +40,65 @@ pub enum Instruction {
     GetPublicKey = 0x04,
     GetScriptSignature = 0x05,
     GetScriptOffset = 0x06,
-    GetMetadataSignature = 0x07,
-    GetScriptSignatureFromChallenge = 0x08,
-    GetViewKey = 0x09,
-    GetDHSharedSecret = 0x10,
+    GetViewKey = 0x07,
+    GetDHSharedSecret = 0x08,
+    GetRawSchnorrSignature = 0x09,
+    GetScriptSchnorrSignature = 0x10,
 }
 
+#[repr(u16)]
+#[derive(FromPrimitive, Debug, Copy, Clone, PartialEq)]
+pub enum AppSW {
```

Comment:
this is delcared twice?

Reply:
Yes, not ideal. To solve this I created a seperate crate for common types shared between the Ledger application and the rest of the code base.